list.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'my_enumerable'
```

Comment:
- [ ] Your enumerable is not correctly imported therefore it throws an error, kindly check the spelling and update it